app/models/registration_history_entry.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class RegistrationHistoryEntry < ActiveRecord::Base
-  has_many :registration_history_change, dependent: :destroy
+  has_many :registration_history_changes, dependent: :destroy
```

Comment:
how does this not break anything?

Reply:
Rails is flexible. It tries both singular and plural for `has_many` associations, but plural is "more correct" in terms of conventions.

Reply:
so add_history_entry will still work even though it does 
`registration_history_change.create`
?

Reply:
No, you have to commit to either singluar or plural.
I CTRL+F'ed through the entire codebase to find all usages of the old singluar `registration_history_change` and replaced them. You will see in the diff for this very PR that `add_history_entry` was also changed accordingly :smile:

Reply:
oh, I missed that